### ISSUE TITLE
Cannot add GitHub Actions build badge

### ISSUE THREAD

**Issue by madskristensen** [microsoft/vsmarketplace#327]
**Describe the bug**
The stock GitHub Actions build badge isn't allowed since it links to an .svg file.

**To Reproduce**
Paste this into the description on the Marketplace page for a VS IDE extension:

`[![Build](https://github.com/madskristensen/EditorColorPreview/actions/workflows/build.yaml/badge.svg)](https://github.com/madskristensen/EditorColorPreview/actions/workflows/build.yaml)`

![image](https://user-images.githubusercontent.com/1258877/153058424-b9213e84-ca85-4d13-b28a-de836a750e21.png)

**Comment by chgill-MSFT** [comment]
Worth noting that, according to the [VS Code docs](https://code.visualstudio.com/api/references/extension-manifest#approved-badges), badges coming from github.com through workflows should work.

I believe @madskristensen also mentioned that this works fine when uploading with the CLI but not when uploading through the website, but he can confirm.

@pratikgawali-msft can you make sure the right person sees this?

**Comment by madskristensen** [comment]
Yes, it works fine from the CLI, so it appears to be a website validation issue only

**Comment by pratikgawali-msft** [comment]
Thanks @madskristensen for reporting this. We have created a bug-item for this in our backlog.
